bundle/src/main/java/com/adobe/acs/commons/redirects/filter/RedirectFilter.java

```diff
@@ -112,7 +112,7 @@
         configurationPolicy = ConfigurationPolicy.REQUIRE, property = {
         SERVICE_DESCRIPTION + "=A request filter implementing support for virtual redirects",
         SLING_FILTER_SCOPE + "=" + EngineConstants.FILTER_SCOPE_REQUEST,
-        SERVICE_RANKING + ":Integer=10000",
+        SERVICE_RANKING + ":Integer=1900",

```

Comment:
Please add a comment here explaining why 1900 is reasonable and what ranking WCMRequestFilter has.